docs/Homebrew-and-Python.md

```diff
@@ -2,18 +2,16 @@
 
 This page describes how Python is handled in Homebrew for users. See [Python for Formula Authors](Python-for-Formula-Authors.md) for advice on writing formulae to install packages written in Python.
 
-Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python.
-
-Homebrew provides formulae to brew Python 3.y. A `python@2` formula was provided until the end of 2019, at which point it was removed due to the Python 2 deprecation.
+Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python. On Linux Homebrew will install the necessary Python version that is needed to make your packages work.
 
 **Important:** If you choose to use a Python which isn't either of these two (system Python or brewed Python), the Homebrew team cannot support any breakage that may occur.
 
 ## Python 3.y
 
-Homebrew provides formulae for maintained releases of Python 3.y (`python@3.y`).
+Homebrew provides formulae for all maintained releases of Python 3.y (`python@3.y`).
```

Comment:
How do you know if a release is maintained or not? Should this be "supported"?

Reply:
Yeah, supported. The support cycle is here, IMO we should link it:  https://devguide.python.org/versions/

Reply:
We only provide these older maintained releases because they are dependencies of other formulae, right? It's not because we *want* to provide **all** of them.